test/controllers/geocoder_controller_test.rb

```diff
@@ -368,17 +368,17 @@ def latlon_check(query, lat, lon)
     assert_template :layout => "map"
     assert_equal %w[latlon osm_nominatim_reverse], assigns(:sources).pluck(:name)
     assert_nil @controller.params[:query]
-    assert_in_delta lat, @controller.params[:lat]
-    assert_in_delta lon, @controller.params[:lon]
+    assert_in_delta lat, @controller.params[:lat].to_f
+    assert_in_delta lon, @controller.params[:lon].to_f
```

Comment:
```suggestion
    assert_match(/^[+-]?\d+(\.\d*)?$/, @controller.params[:lat])
    assert_match(/^[+-]?\d+(\.\d*)?$/, @controller.params[:lon])
    assert_in_delta lat, @controller.params[:lat].to_f
    assert_in_delta lon, @controller.params[:lon].to_f
```

Reply:
Nice idea. I've added that and also added a test case using small values to check it doesn't trigger scientific formatting.